docs/llm/reference.md

```diff
@@ -161,6 +161,12 @@ convert_tokenizer -o {target folder name} --utf8_replace_mode replace --with-det
 
 Check [tested models](https://github.com/openvinotoolkit/openvino.genai/blob/master/tests/python_tests/models/real_models).
 
+## Input preprocessing
+
+When loading the pipeline, model server loads not only model files, but also tokenizer configuration present in `tokenizer_config.json`. Make sure your tokenizer configuration is correct - results accuracy might depend on it. Also note that `tokenizer_config.json` with fields filled with incorrect datatypes (for example `"bos_token": {"key": "val"}`) **will not work at all and the pipeline with such tokenizer config will not load**. 
```

Comment:
I'm not sure if this comment is relevant while this file is generated automatically by optimum-intel package

Reply:
In general yes, but this part describes serving behavior, so I think it should be kept here as a reference of what to expect.